### PR TITLE
Remove checkout zones

### PR DESCRIPTION
**Pull request by mbj**

### CODE REVIEW

**Comment by dkubb** [comment]
@mbj I will merge this when green.

File: core/spec/helpers/base_helper_spec.rb [hunk 1]
```diff
@@ -3,48 +3,13 @@
 
   let(:current_store) { create(:store) }
 
-  context 'available_countries' do
-    let(:country) { create(:country) }
-
+  context '#available_countries' do
     before do
       3.times { create(:country) }
```
Comment (dkubb) [inline core/spec/helpers/base_helper_spec.rb]: What if we changed this to:

```ruby
let(:countries) { 3.times.map { create(:country) } }
```

And then below we can do:

```ruby
expect(available_countries.to_a).to eql(countries)
```

File: core/spec/helpers/base_helper_spec.rb [hunk 2]
```diff
@@ -3,48 +3,13 @@
...
-          state_zone.members.create!(:zoneable => state)
-          Spree::Config[:checkout_zone] = state_zone.name
-        end
-
-        it "return complete list of countries" do
-          expect(available_countries.count).to eq(Spree::Country.count)
-        end
-      end
+    it 'returns countries in expected order' do
+      expect(available_countries.to_a).to eql(Spree::Country.order(:id).to_a)
```
Comment (dkubb) [inline core/spec/helpers/base_helper_spec.rb]: You should probably be able to remove the `#to_a` from both sides of the equation here, although I'd probably recommend trying https://github.com/MountainRoseHerbs/spree/pull/272/files#r62371602 first.

Comment (mbj) [inline core/spec/helpers/base_helper_spec.rb]: No, the `#to_a` is required as equality on AR relations uses object identity.

Comment (mbj) [inline core/spec/helpers/base_helper_spec.rb]: And if I take your suggestion at least "one" can be removed, I'll do that.

**Pull request merged**